falcon-tracer/conf/logging.ini

```diff
@@ -2,21 +2,27 @@
 keys=root
 
 [handlers]
-keys=consoleHandler
+keys=consoleHandler,fileHandler
 
 [formatters]
 keys=simpleFormatter
 
 [logger_root]
 level=DEBUG
-handlers=consoleHandler
+handlers=consoleHandler,fileHandler
 
 [handler_consoleHandler]
 class=StreamHandler
 level=DEBUG
 formatter=simpleFormatter
-args=(sys.stderr,)
+args=(sys.stdout,)
+
+[handler_fileHandler]
+class=FileHandler
+level=DEBUG
+formatter=simpleFormatter
+args=('falcon_tracer.log',)
 
 [formatter_simpleFormatter]
-format=[%(asctime)s][%(levelname)s %(filename)s(line %(lineno)s)] - %(message)s
```

Comment:
It is better to create a new formatter `eventFormatter` for events. Otherwise usual logs will lack relevant information.